hbw/app/controllers/hbw/tasks_controller.rb

```diff
@@ -18,6 +18,7 @@ def index
 
     def claiming
       @tasks = widget.claiming_task_list(current_user_identifier, entity_class, assigned, max_results, search_query)
+      head :no_content
```

Comment:
We return JSON with tasks, why `no_content`?

Reply:
Oops, wrong place :(